### DISCUSSION TITLE
稳定版（7.8.6）双击直接闪退，测试版找不到登录的入口，怎么弄？

### DISCUSSION THREAD

**Discussion by AntonioCXC** [getlantern/lantern-forum-cn#631] [General]
<img width="258" alt="20240621222844" src="https://github.com/getlantern/lantern-forum-cn/assets/173478572/1ebd986d-5f58-4fbe-be93-9bd922c0211d">

不知道为什么，重装了一次系统，然后稳定版的7.8.6版安装以后，双击打开直接闪退，右下角连托盘图标都出不来，后台的lantern.exe进程也是自动就没了。
没办法，只能用测试版，好像是8.0.0还是啥？但为啥连个账户登录的入口都没有啊？我之前的账户里还有挺长时间的VIP呢，不能接着用？非得重新再买一次？
麻烦告知一下怎么解决。

P.S. 我的专业版账户邮箱是 cxc0715@foxmail.com
又P.S. 我看到7.8.8版本的更新日志了，但是，没有哪个链接能下载7.8.8版本，首页几个链接下载回来的都是7.8.6，也就是闪退的版本

**Comment by 904787103** [comment]
同样问题，正式版打开闪退，8.0版本打开无法授权，花了钱完全用不了

**Comment by MaxMKMax** [comment]
@AntonioCXC @904787103 

您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by AntonioCXC** [reply]
就，首先，都说了7.8.6是闪退了，进都进不去，谈何“在应用程序中提交报告”？
然后，刚刚8.0.0测试版的“Report Issue”功能我也看了，就是个摆设，点“发送报告”直接报错。

**Comment by tena1210** [comment]
你们每次发布应用，自己试一遍呢，重新卸载再安装一遍，每次都要折腾我们，I 服了 U。。。
